modin/modin_logging/logger_metaclass.py

```diff
@@ -0,0 +1,93 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+from functools import wraps
+from .config import get_logger
+from types import FunctionType, MethodType
+from modin.config import LogMode
```

Comment:
```suggestion
from functools import wraps
from types import FunctionType, MethodType

from .config import get_logger
from modin.config import LogMode
```
Third party imports go first and Modin ones go next.